packages/shared/test-suites/RecordingTestMetadata.ts

```diff
@@ -1076,3 +1082,13 @@ export function getTestEnvironment(groupedTestCases: AnyGroupedTestCases): TestE
   }
   return groupedTestCases.environment;
 }
+
+export function getTestRunId(groupedTestCases: AnyGroupedTestCases): string | null {
+  if (isGroupedTestCasesV1(groupedTestCases)) {
+    return null;
+  } else if (isGroupedTestCasesV2(groupedTestCases)) {
+    return groupedTestCases.run.id;
+  } else {
+    return groupedTestCases.runId;
+  }
+}
```

Comment:
@ryanjduffy Is this  safe to rely on?

Reply:
I think every recording with any test metadata will have a test run id (in `metadata.test.run.id`).